include/CL/cl.h

```diff
@@ -1375,6 +1375,18 @@ clEnqueueTask(cl_command_queue  /* command_queue */,
               const cl_event *  /* event_wait_list */,
               cl_event *        /* event */) CL_EXT_SUFFIX__VERSION_1_2_DEPRECATED;
 
+/* Poclcc OpenCL APIs */
+extern CL_API_ENTRY cl_int CL_API_CALL
+clExportPoclccBinary(cl_program /* program */,
+                     void **    /* poclcc_binary */,
+                     cl_uint *  /* poclcc_binary_size */);
+  
+extern CL_API_ENTRY cl_int CL_API_CALL
+clExtractPoclccBinary(void *            /* input_buffer */,
+                      cl_uint           /* input_buffer_size */,
+                      size_t **         /* lengths */,
+                      unsigned char *** /* binaries */);
+
```

Comment:
cl.h should not be modified as it's provided by Khronos. 
An extension file should be added for pocl in cl/cl_pocl.h